copyright

```diff
@@ -490,6 +490,24 @@ Copyright: Becca Tommaso (tommasobecca03@gmail.com)
 License: CC-BY-SA-4.0
 Comment: Derived from works by Michael Zahniser (under the same license).
 
+Files
+ sounds/pincer*
+Copyright: Nomadic Volcano (nomadicvolcano@gmail.com)
+License: GPL 2.0
```

Comment:
```suggestion
License: GPL-2
```
The field cannot have the space character.

This PR has to contain the entire text of GPL version 2, or... I'm not sure what Nomadic Volcano altered the sounds, but probably they come from [skeleton-big-hit-3.ogg](https://github.com/wesnoth/wesnoth/blob/master/data/core/sounds/skeleton-big-hit-3.ogg) and [petrified.ogg](https://github.com/wesnoth/wesnoth/blob/master/data/core/sounds/petrified.ogg) in [data/core/sounds](https://github.com/wesnoth/wesnoth/tree/master/data/core/sounds) of [The Buttle for Wesnoth](https://github.com/wesnoth/wesnoth). If you recreate your versions from skeleton-big-hit-3.ogg and petrified.ogg, you can publish them under GPL-3 or GPL-3+ on your responsibility.